src/module.ts

```diff
@@ -26,6 +26,7 @@ export default defineNuxtModule<ModuleOptions>({
     rootDir: nuxt.options.rootDir,
     componentDirs: [],
     components: [],
+    sources: [],
```

Comment:
I think using `metaSources` might be a better name for this option and will be consistent with `metaFields` 

Also, CLI should generate all `metaFields` and module can filter them before using them.